tests/test_compute_funs.py

```diff
@@ -1579,55 +1579,131 @@ def test_contravariant_basis_vectors_PEST():
+        "g_rr",
+        "g_rt",
+        "g_rz",
+        "g_tt",
+        "g_tz",
+        "g_zz",
+        "g^rt",
+        "g_rr_t",
+        "g_rr_z",
+        "g_tt_r",
+        "g_tt_z",
+        "g_zz_t",
+        "g_rt_z",
+        "g^rr_z",
+        "g^rr_t",
+        "g^rt_z",
+        "g^rt_t",
+        "g^rz_z",
+        "g^rz_t",
+        "sqrt(g)_r",
+        "sqrt(g)_t",
+        "sqrt(g)_z",
+    ]
+    keys_PEST = [
+        "(e^rho_v)|PEST",
+        "(e^rho_p)|PEST",
+        "e^vartheta",
+        "(e^vartheta_v)|PEST",
+        "(e^vartheta_p)|PEST",
+        "(e^zeta_v)|PEST",
+        "(e^zeta_p)|PEST",
+        "(e_theta_PEST_v)|PEST",
+        "(e_theta_PEST_p)|PEST",
+        "(e_phi_p)|PEST",
+        "(e_theta_PEST_r)|PEST",
+        "(e_phi_r)|PEST",
+        "(e_rho_r)|PEST",
+        "J^theta_PEST",
+        "(J^theta_PEST_v)|PEST",
+        "(J^theta_PEST_p)|PEST",
+        "(J^zeta_v)|PEST",
+        "(J^zeta_p)|PEST",
+        "g_rr|PEST",
+        "g_rv|PEST",
+        "g_rp|PEST",
+        "g_vv|PEST",
+        "g_vp|PEST",
+        "g_pp|PEST",
+        "g^rv",
+        "(g_rr_v)|PEST",
+        "(g_rr_p)|PEST",
+        "(g_vv_r)|PEST",
+        "(g_vv_z)|PEST",
+        "(g_pp_v)|PEST",
+        "(g_rv_p)|PEST",
+        "(g^rr_p)|PEST",
+        "(g^rr_v)|PEST",
+        "(g^rv_p)|PEST",
+        "(g^rv_v)|PEST",
+        "(g^rz_p)|PEST",  # TODO: fix name
+        "(g^rz_v)|PEST",  # TODO: fix name
+        "(sqrt(g)_PEST_r)|PEST",
+        "(sqrt(g)_PEST_v)|PEST",
+        "(sqrt(g)_PEST_p)|PEST",
+    ]
+    index = data_index["desc.equilibrium.equilibrium.Equilibrium"].keys()
+    keys_DESC, keys_PEST = zip(
+        *[(d, p) for d, p in zip(keys_DESC, keys_PEST) if (d in index) and (p in index)]
+    )
+    keys_DESC = list(keys_DESC)
+    keys_PEST = list(keys_PEST)
 
-    for eq in eq_list:
-        eq = to_sfl(eq)
-
-        # Add more quantities to check
-        keys_DESC = [
-            "J^theta",
-            "J^theta_t",
-            "J^theta_z",
-            "sqrt(g)_t",
-            "sqrt(g)_z",
-            "g_tt",
-            "g_rr",
-            "g_zz",
-            "g_tt_r",
-            "g^rr_t",
-            "g^rr_z",
-        ]
-        keys_PEST = [
-            "J^theta_PEST",
-            "(J^theta_PEST_v)|PEST",
-            "(J^theta_PEST_p)|PEST",
-            "(sqrt(g)_PEST_v)|PEST",
-            "(sqrt(g)_PEST_p)|PEST",
-            "g_vv|PEST",
-            "g_rr|PEST",
-            "g_pp|PEST",
-            "(g_vv_r)|PEST",
-            "(g^rr_v)|PEST",
-            "(g^rr_p)|PEST",
-        ]
+    grid = LinearGrid(
+        rho=np.linspace(0.2, 1, 10), M=eq.M_grid, N=eq.N_grid, NFP=eq.NFP, sym=eq.sym
+    )
+    data = eq_PEST.compute(keys_DESC + keys_PEST, grid)
+    np.testing.assert_allclose(data["theta_PEST_t"], 1)
+    data_to_verify = eq.compute(keys_PEST, grid)
 
-        data_DESC = eq.compute(keys_DESC)
-        data_PEST = eq.compute(keys_PEST)
```

Comment:
The same equilibrium was being used to compute this stuff, so the test was not doing anything.